filebeat/fileset/fileset.go

```diff
@@ -37,7 +37,6 @@ func New(
 	name string,
 	mcfg *ModuleConfig,
 	fcfg *FilesetConfig) (*Fileset, error) {
-
 	modulePath := filepath.Join(modulesPath, mcfg.Module)
```

Comment:
I agree, I think in cases where the function declaration spans multiple lines I will leave those blank lines alone. Will update.